core/storage/user/gae_models_test.py

```diff
@@ -2245,6 +2245,94 @@ def test_get_translation_reviewer_user_ids(self):
         self.assertTrue(self.USER_ID_1 in translation_reviewer_ids)
         self.assertTrue(self.USER_ID_2 in translation_reviewer_ids)
 
+    def test_get_num_reviewers_for_translations_in_lang_code_if_no_reviewers(
```

Comment:
Similarly to an earlier comment: what exactly is the expected outcome?

Ditto for below test names.